Automattic/woocommerce-payments
# Update CSV export language codes to align with the languages supported by the server

#### Changes proposed in this Pull Request
At present, the default WordPress locale codes do not align with the supported languages on the server. This PR modifies the WordPress locale codes to match the supported languages. 

Additionally, it hides the CSV export modal if the language is not supported.

#### Testing instructions
*Case 1*
- Go to `Settings` -> `General`.
- Set the site language to `Spanish`.
- Go to `Payments` -> `Settings`.
- Scroll down to the Reporting section.
- Confirm that the export language selection box is displayed.

*Case 2*
- Go to `Settings` -> `General`.
- Set the site language to `Bulgarian`.
- Go to `Payments` -> `Settings`.
- Confirm that the Report section is not displayed.

*Case 3*
- Go to `Settings` -> `General`.
- Set the site language to `Spanish`.
- Go to `Payments` -> `Transactions`.
- Click the Download button on the top right corner of the table.
- A modal will be displayed prompting to select the download language.
- Uncheck the `Remember the language` settings checkbox.
- Select `Site language `-> `Spanish` and click download.
- Verify the `locale` param in the request is `es`.

-------------------


**Post merge**
